src/ledger/cleanup_service.zig

```diff
@@ -31,19 +31,20 @@ const DEFAULT_CLEANUP_SLOT_INTERVAL: u64 = 512;
 const LOOP_LIMITER = Duration.fromMillis(DEFAULT_CLEANUP_SLOT_INTERVAL * DEFAULT_MS_PER_SLOT / 10);
 
 pub fn run(
-    logger: sig.trace.Logger,
+    logger_: sig.trace.Logger,
     blockstore_reader: *BlockstoreReader,
     db: *BlockstoreDB,
     lowest_cleanup_slot: *sig.sync.RwMux(Slot),
     max_ledger_shreds: u64,
     exit: *AtomicBool,
 ) !void {
+    const logger = logger_.withScope(@src().fn_name);
     var last_purge_slot: Slot = 0;
 
     logger.info().log("Starting blockstore cleanup service");
     while (!exit.load(.acquire)) {
         last_purge_slot = try cleanBlockstore(
-            logger,
+            logger.unscoped(),
```

Comment:
This file can use a single scope.

Reply:
That's not necessary. You can just specify the same scope in both places. 
```zig
logger.withScope("ledger.cleanup_service");
...
logger_: sig.trace.ScopedLogger("ledger.cleanup_service"),
```
Or you can declare a const for the scoped logger
```zig
const Logger = sig.trace.ScopedLogger("ledger.cleanup_service");
```

Reply:
[3b7bc77](https://github.com/Syndica/sig/pull/345/commits/3b7bc77f96f10e2a94dd07636156730e3e8e6640)